modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc

```diff
@@ -123,6 +147,106 @@ STObstaclesProcessor::GetAllSTBoundaries() {
   return obs_id_to_st_boundary_;
 }
 
+bool STObstaclesProcessor::GetSBoundsFromDecisions(double t,
+    std::vector<std::pair<double, double>>* const available_s_bounds,
+    std::vector<std::vector<std::pair<std::string, ObjectDecisionType>>>*
+        const available_obs_decisions) {
+  // Sanity checks.
+  CHECK_NOTNULL(available_s_bounds);
+  CHECK_NOTNULL(available_obs_decisions);
```

Comment:
let's avoid CHECK in runtime function. only exception to use CHECK is in init function. thanks.

Reply:
Thanks. I fixed this.